store/server/features/employees/queries.ts

```
@@ -0,0 +1,92 @@
+import { useQuery } from 'react-query';
+import { useAuthenticationStore } from '@/store/uistate/features/authentication';
+import { ORG_AND_EMP_URL } from '@/utils/constants';
+import { crudRequest } from '@/utils/crudRequest';
+import { requestHeader } from '@/helpers/requestHeader';
+
+const getHeaders = async () => {
+  const headers = await requestHeader();
+  console.log('Headers from requestHeader:', headers);
```

ESLint

[error] Unexpected console statement.

(no-console)